src/Twee/School/Bribe Teacher.twee

```diff
@@ -24,17 +24,17 @@ She takes a quick glance at how much money you have, but just laughs coldly.
 
 You hurriedly put your wallet away.
 
-@@.teacher;"Now, $player.name, I have given a lot of thought to what you tried earlier. I am insulted you think you could buy your way out of trouble with some pocket change. 
+@@.teacher;"Now, $player.name, I have given a lot of thought to what you tried earlier. I am insulted you think you could buy your way out of trouble with some pocket change."@@
 	
-"If money is what is important to you, then perhaps I can use it to teach you a lesson. If you wish to prevent me from reporting you to the principal, it will cost you $<<print $bribeAmount>> every Monday.\
+\
 <<if $player.bribeDiscount gt 0>>\
-	Since it's already <<print window.timeCode.weekDayText()>>, I will reduce this week's amount by $<<print $player.bribeDiscount>>, to $<<print _nb>>.@@
+	@@.teacher;"If money is what is important to you, then perhaps I can use it to teach you a lesson. If you wish to prevent me from reporting you to the principal, it will cost you $<<print $bribeAmount>> every Monday. Since it's already <<print window.timeCode.weekDayText()>>, I will reduce this week's amount by $<<print $player.bribeDiscount>>, to $<<print _nb>>."@@
 	
 <<endif>>\
 <<if $flags.bullySteal>>\
-	"I will also stop those $futa.boy_girl////s from stealing your money in the morning.@@
+	@@.teacher;"If money is what is important to you, then perhaps I can use it to teach you a lesson. If you wish to prevent me from reporting you to the principal, it will cost you $<<print $bribeAmount>> every Monday. I will also stop those $futa.boy_girl////s from stealing your money in the morning."@@
```

Comment:
It's not mutally exclusive that the player will have a bribe discount and have their money stolen that day.  In fact, it's likely both will be needed. That's why these two parts are joined like they are.
```suggestion
	@@.teacher;I will also stop those $futa.boy_girl////s from stealing your money in the morning.@@
```